orchagent/natorch.cpp

```diff
@@ -1720,6 +1729,7 @@ bool NatOrch::removeHwSnaptEntry(const NaptEntryKey &keyEntry)
     }
     deleteNaptCounters(keyEntry.prototype.c_str(), keyEntry.ip_address, keyEntry.l4_port);
     m_naptEntries.erase(keyEntry);
+    gCrmOrch->decCrmResUsedCounter(CrmResourceType::CRM_SNAT_ENTRY);
```

Comment:
Current Sonic CrmOrch code doesn't have support for double_nat. When we add that support, we can update crm counters.

Reply:
Yes, not yet supporting the double nat.